tutor/templates/build/openedx/Dockerfile

```diff
@@ -190,15 +180,9 @@ ENV REVISION_CFG /openedx/config/revisions.yml
 COPY --chown=app:app settings/lms/*.py ./lms/envs/tutor/
 COPY --chown=app:app settings/cms/*.py ./cms/envs/tutor/
 
-# Copy user-specific locales to /openedx/locale/user/locale and compile them
-RUN mkdir /openedx/locale/user
-COPY --chown=app:app ./locale/ /openedx/locale/user/locale/
-RUN cd /openedx/locale/user && \
-    django-admin compilemessages -v1
-
-# Compile i18n strings: in some cases, js locales are not properly compiled out of the box
-# and we need to do a pass ourselves. Also, we need to compile the djangojs.js files for
-# the downloaded locales.
+# Pull latest translations via atlas
+RUN atlas pull {{ ATLAS_OPTIONS }} translations/edx-platform/conf/locale:conf/locale
```

Comment:
I think that ATLAS_OPTIONS should be empty by default, and we should explicitely pass `--repository='{{ ATLAS_REPOSITORY }}' --branch '{{ ATLAS_REVISION }}'` to all `atlas pull` commands. Like so:

    RUN atlas pull --repository='{{ ATLAS_REPOSITORY }}' --branch '{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }} translations/edx-platform/conf/locale:conf/locale

Otherwise, users need to be smart about properly setting the upstream repository whenever they want to customise atlas options.

Reply:
seems appropriate. done.